### ISSUE TITLE
Program crashes while checking in at all since latest commit

### ISSUE THREAD

**Issue by ersan** [mwynholds/southy#3]
I get this when I add a flight that is already available for checkin (i.e. past the checkin time already):

```
ERROR  2016-03-16 19:26:03  Unexpected error
undefined method `upcase' for nil:NilClass
Stacktrace:
/home/ersan/.rbenv/versions/1.9.3-p448/lib/ruby/gems/1.9.1/gems/southy-1.2/lib/southy/config.rb:85:in `block in remove'
/home/ersan/.rbenv/versions/1.9.3-p448/lib/ruby/gems/1.9.1/gems/southy-1.2/lib/southy/config.rb:84:in `delete_if'
/home/ersan/.rbenv/versions/1.9.3-p448/lib/ruby/gems/1.9.1/gems/southy-1.2/lib/southy/config.rb:84:in `remove'
/home/ersan/.rbenv/versions/1.9.3-p448/lib/ruby/gems/1.9.1/gems/southy-1.2/lib/southy/travel_agent.rb:29:in `block in confirm'
/home/ersan/.rbenv/versions/1.9.3-p448/lib/ruby/gems/1.9.1/gems/southy-1.2/lib/southy/travel_agent.rb:23:in `each'
/home/ersan/.rbenv/versions/1.9.3-p448/lib/ruby/gems/1.9.1/gems/southy-1.2/lib/southy/travel_agent.rb:23:in `confirm'
/home/ersan/.rbenv/versions/1.9.3-p448/lib/ruby/gems/1.9.1/gems/southy-1.2/lib/southy/daemon.rb:46:in `block in run'
/home/ersan/.rbenv/versions/1.9.3-p448/lib/ruby/gems/1.9.1/gems/southy-1.2/lib/southy/daemon.rb:45:in `each'
/home/ersan/.rbenv/versions/1.9.3-p448/lib/ruby/gems/1.9.1/gems/southy-1.2/lib/southy/daemon.rb:45:in `run'
/home/ersan/.rbenv/versions/1.9.3-p448/lib/ruby/gems/1.9.1/gems/southy-1.2/lib/southy/daemon.rb:24:in `start'
/home/ersan/.rbenv/versions/1.9.3-p448/lib/ruby/gems/1.9.1/gems/southy-1.2/lib/southy/service.rb:21:in `block in start'
/home/ersan/.rbenv/versions/1.9.3-p448/lib/ruby/gems/1.9.1/gems/southy-1.2/lib/southy/service.rb:21:in `fork'
/home/ersan/.rbenv/versions/1.9.3-p448/lib/ruby/gems/1.9.1/gems/southy-1.2/lib/southy/service.rb:21:in `start'
/home/ersan/.rbenv/versions/1.9.3-p448/lib/ruby/gems/1.9.1/gems/southy-1.2/lib/southy/cli.rb:20:in `start'
/home/ersan/.rbenv/versions/1.9.3-p448/lib/ruby/gems/1.9.1/gems/southy-1.2/bin/southy:54:in `<top (required)>'
/home/ersan/.rbenv/versions/1.9.3-p448/bin/southy:23:in `load'
/home/ersan/.rbenv/versions/1.9.3-p448/bin/southy:23:in `<main>'
```

Haven't been able to test something added prior to checkin and then waiting for checkin time to happen.

**Comment by ersan** [comment]
This seems to be a problem adding any flight on 11f1748 - the commit right before it (2632f44) still works fine.

**Comment by ersan** [comment]
Fixed in 65e0b51
